docs/sources/clients/_index.md

```diff
@@ -2,7 +2,7 @@
 title: Clients
 weight: 600
 ---
-# Grafana Loki clients
+# Grafana Loki clients for sending data to Loki
```

Comment:
Since this is the title of the page, I would rather make a new sub-heading called `Clients for sending data`. Same for `Clients for reading data`. Wdyt?

Reply:
I think we can leave the heading "Grafana Loki Clients" since the first sentence under the heading is "Grafana Loki supports the following official clients for sending logs:"

Having "for sending data" in both the heading and the first sentence feels repetitive to me.